src/opnsense/scripts/unbound/wrapper.py

```diff
@@ -42,6 +42,12 @@ def unbound_control_reader(action):
         for line in output_stream:
             yield line.decode()
 
+try:
+    if subprocess.call("pgrep -F /var/run/unbound.pid 2>/dev/null >/dev/null", shell=True) != 0:
```

Comment:
I would go for a more native python approach here, for inspiration, this is how the people from psutil seem to fix this : https://github.com/giampaolo/psutil/blob/c10df5aa04e1ced58d19501fa42f08c1b909b83d/psutil/_psposix.py#L32-L51